test/test_util_http.cpp

```diff
@@ -377,3 +377,138 @@ TEST(HTTPParser_ParseHeaderLine)
         }
     }
 }
+
+template <typename Socket>
+struct ChunkedEncodingHTTPParser : public HTTPParser<Socket> {
+    StringData key;
+    StringData value;
+    std::string body;
+    std::error_code error;
+
+    ChunkedEncodingHTTPParser(Socket& socket, const std::shared_ptr<util::Logger>& logger_ptr)
+        : HTTPParser<Socket>{socket, logger_ptr}
+    {
+        this->m_has_chunked_encoding = true;
```

Comment:
nit: no need to specify `this->` when referencing a member variable.

Reply:
Because `ChunkedEncodingHTTPParser` is template dependant `this->` is required to access the member variable.